railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -21,6 +21,11 @@ source 'https://rubygems.org'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+<%- if options.api? -%>
+# Use ActiveModelSerializers to serialize JSON responses
+gem 'active_model_serializers', '~> 0.10.0.rc1'
```

Comment:
@yuki24 the important thing to me is to be able to quickly switch into both.
The good thing AMS have is that you don't want to implement every single serialization point. You just define the serializers and relations and you don't have to build (and possibly make a mistake) on every single possible result. And even more you could switch formats just with a configuration.